static/builder/common.js

```diff
@@ -430,6 +430,16 @@ function innerCalculateBuildValueWithFormula(itemAndPassives, unitBuild, ennemyS
                 "max": (baseDamage * variance.max + magDamage) * context.damageMultiplier.max / 2,
                 "switchWeapons": switchWeapons
             }   
+        } else if(formula.value.mecanism == "summonerSkill"){
```

Comment:
There is no need for a "summonerSkill" section here, it should already be taken care of by the damage part of the calculation